scripts/bundle_clusterfuzz.py

```diff
@@ -87,7 +88,9 @@
     # Delete the argument, as importing |shared| scans it.
     sys.argv.pop()
 
+from test import fuzzing # noqa
 from test import shared # noqa
+from test import support # noqa
```

Comment:
What lints are we skipping here?

Reply:
The one that imports must be on top (which we can't do because of lines 83-89, which I can't think of a nice alternative to).